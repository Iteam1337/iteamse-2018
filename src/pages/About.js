// @flow

import * as React from 'react'
import * as Iteam from '../typings/iteam.flow'
import * as IteamCMS from './__generated__/AboutPage'
import { Query } from 'react-apollo'
import gql from 'graphql-tag'
import Header from '../components/Header/Header'
import aboutUsFullImage from './img/bleed_aboutus.jpg'
import aboutUsImage from './img/block_aboutus.png'
import Block from '../components/Blocks/Block'
import ImageBleed from '../components/Blocks/ImageBleed'
import ImageBlock from '../components/Blocks/ImageBlock'
import styled from 'styled-components'

type Props = Iteam.ApolloBase<IteamCMS.AboutPage>

const AboutPageQuery = gql`
  query AboutPage {
    pageAboutUs {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      funText
      funTitle
      goodText
      goodTitle
      stabilityText
      stabilityTitle
      valueText
      valueTitle
    }
  }
`

const Content = styled.div`
  display: grid;
  grid-row-gap: 40px;
  padding-bottom: 40px;
  padding-top: 40px;

  @media (min-width: 1024px) {
    grid-row-gap: 100px;
    padding-bottom: 100px;
    padding-top: 100px;
  }
`

const About = () => {
  return (
    <Query query={AboutPageQuery}>
      {({ loading, data: { pageAboutUs } }: Props) => {
        if (loading) {
          return null
        }

        return (
          <React.Fragment>
            <Header
              backgroundImage={pageAboutUs.headerImage}
              messageBgColor={pageAboutUs.headerTextBgColor}
              messageOne={pageAboutUs.headerText1}
              messageTwo={pageAboutUs.headerText2}
            />
            <Content>
              <Block title={pageAboutUs.valueTitle}>
                {pageAboutUs.valueText}
              </Block>
              <Block title={pageAboutUs.funTitle}>{pageAboutUs.funText}</Block>
              <ImageBlock image={aboutUsImage} />
              <Block title={pageAboutUs.goodTitle}>
                {pageAboutUs.goodText}
              </Block>
              <ImageBleed image={aboutUsFullImage} />
              <Block title={pageAboutUs.stabilityTitle}>
                {pageAboutUs.stabilityText}
              </Block>
            </Content>
          </React.Fragment>
        )
      }}
    </Query>
  )
}

export default About
