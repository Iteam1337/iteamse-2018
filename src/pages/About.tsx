import gql from 'graphql-tag'
import * as React from 'react'
import { Query } from 'react-apollo'
import { AboutPageQuery } from '../../typings/iteamse'
import Block from '../components/Blocks/Block'
import ImageBleed from '../components/Blocks/ImageBleed'
import ImageBlock from '../components/Blocks/ImageBlock'
import GridColumn from '../components/Grid/GridColumn'
import Header from '../components/Header/Header'
import Team from '../components/Team/Team'
import styled from '../theme'
import aboutUsImage from './img/illustrations-group-1.svg'

export const ABOUT_PAGE_QUERY = gql`
  query AboutPage {
    pageAboutUs {
      headerImage
      headerText1
      headerText2
      headerTextBgColor
      funText
      funTitle
      goodText
      goodTitle
      imageBleed
      stabilityIcons
      stabilityText
      stabilityTitle
      valueText
      valueTitle
    }
  }
`

const StabilityIcons = styled.div`
  align-items: center;
  display: flex;
  margin-top: 40px;
`

const StabilityIcon = styled.img`
  max-width: 145px;

  &:not(:last-child) {
    margin-right: 20px;
  }
`

class AboutQuery extends Query<AboutPageQuery> {}

const About = () => {
  return (
    <AboutQuery query={ABOUT_PAGE_QUERY}>
      {({ loading, data }) => {
        if (loading || !data) {
          return null
        }

        const { pageAboutUs } = data

        return (
          <>
            <Header
              backgroundImage={pageAboutUs.headerImage}
              messageBgColor={pageAboutUs.headerTextBgColor}
              messageOne={pageAboutUs.headerText1}
              messageTwo={pageAboutUs.headerText2}
            />

            <GridColumn>
              <Block title={pageAboutUs.valueTitle}>
                {pageAboutUs.valueText}
              </Block>
              <Block title={pageAboutUs.funTitle}>{pageAboutUs.funText}</Block>
              <ImageBlock image={aboutUsImage} />
              <Block title={pageAboutUs.goodTitle}>
                {pageAboutUs.goodText}
              </Block>
              <ImageBleed image={pageAboutUs.imageBleed} />
              <Block title={pageAboutUs.stabilityTitle}>
                {pageAboutUs.stabilityText}

                <StabilityIcons>
                  {pageAboutUs.stabilityIcons.map(icon => {
                    return typeof icon === 'string' ? (
                      <StabilityIcon alt="" key={icon} src={icon} />
                    ) : null
                  })}
                </StabilityIcons>
              </Block>
            </GridColumn>

            <Team bgColor="green" shortName={['rln', 'msr']} />
          </>
        )
      }}
    </AboutQuery>
  )
}

export default About
