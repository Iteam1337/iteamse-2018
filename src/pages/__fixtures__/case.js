export const workCase = {
  headerBgColor: '#1616B2',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  slug: 'arbetsformedlingen',
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  title: 'Arbetsförmedlingen',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  __typename: 'Case',
}
