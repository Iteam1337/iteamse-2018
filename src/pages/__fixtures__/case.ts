export const workCase = {
  __typename: 'Case',
  aboutCompany:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  aboutCompanyTitle: 'Arbetsförmedlingen',
  contact:
    '- [Hans Rollman](mailto:hans.rollman@iteam.se)\n- Johan Littorin, Telenor, 0734 41 72 85',
  contactTitle: 'Presskontakt',
  development:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  developmentTitle: 'Utveckling',
  headerBgColor: '#1616B2',
  introduction:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  introductionTitle: 'En ny digital tjänst',
  logo:
    '//images.ctfassets.net/rj4r6yfcesw5/4p0ptdLj9SaECekOI4c6C2/c2cdeab1cebc9650ebb0e7778357c231/AF_ordbild_vit.png',
  partners: '[Digitalbyrån](https://hej.com)',
  partnersTitle: 'Samarbetspartners',
  process:
    'Aeneanurutrum, erat at tempus porttitor, mi eros vestibulum magna, eget semper mauris ipsum a nisi. Interdum et malesuada fames ac ante ipsum primis in faucibus. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Cras quis odio vel turpis pellentesque finibus. Pellentesque cursus diam ac ligula placerat volutpat. Vivamus dictum augue eu rutrum efficitur. Suspendisse ut pellentesque ante. Suspendisse odio velit, fringilla eu porta vel, elementum ac libero. Nam eu vestibulum magna. Vestibulum quis risus elementum, lobortis sapien at, rhoncus enim. Nam accumsan odio lacus, at facilisis lacus rhoncus vel. Suspendisse sollicitudin a ipsum at cursus.',
  processTitle: 'Process',
  quote:
    '"We feel that our crew at Iteam are more colleagues than suppliers. Rather than specifying a tech spec to work on, we collaborate, carefully chiseling out functionality as we go along. This way of working has minimized waste and it has also taken us further than we ever imagined”',
  quotePerson: 'Anna Andersson, chef Arbetsförmedlingen',
  slug: 'arbetsformedlingen',
  tags: ['API', 'Docker', 'UX', 'React', 'Vue'],
  team: null,
  thumbnailImage:
    '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
  title: 'Arbetsförmedlingen',
}
