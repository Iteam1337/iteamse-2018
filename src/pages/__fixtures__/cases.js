export const cases = [
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'vimla',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/68OlBclfQAGMcEUsQgcIUY/9bdf52bb8a9a17be06e11ef3fa32d3de/vimla-case-phone.png',
    title: 'Vimla',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'tsab',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/UGucAeFycSqIGw0CAc0ys/3d58300261bcff64dba751de9228a150/tsab-case-phone.png',
    title: 'Taxi Stockholm',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'seb',
    thumbnailImage: null,
    title: 'SEB',
    __typename: 'Case',
  },
  {
    location: 'Stockholm',
    shortDescription:
      'Synliggörande av asylsökandes kompetens med digitala verktyg',
    slug: 'arbetsformedlingen',
    thumbnailImage:
      '//images.ctfassets.net/rj4r6yfcesw5/3Y7zZ25WbeOCU8w84kAEos/b97ff1899cf6f2e4dca0eb2b4f0d5a44/af.png',
    title: 'Arbetsförmedlingen',
    __typename: 'Case',
  },
]
