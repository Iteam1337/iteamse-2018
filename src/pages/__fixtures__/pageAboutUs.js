export const pageAboutUs = {
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/j8ng4HkakgMCyIMqeek2A/82be3a1d363b21dd29b081e76d0fdfeb/about.jpg',
  headerText1: 'Vi skapades ur en vilja att skapa värde,',
  headerText2: 'ha kul och göra något bra',
  headerTextBgColor: '#54FBAC',
  funText:
    'Vi utvecklar din tjänst tillsammans med dig. Det blir mycket bättre så. Och roligare. För oss är det viktigt att ha kul. Därför väljer vi uppdrag med omsorg. Det blir liksom ett löfte till våra kunder: “Vi lovar att bara jobba med dig så länge vi tycker det är kul och brinner för det du gör”. Det lovar vi därför att då blir resultatet bättre.',
  funTitle: 'Ha kul',
  goodText:
    'Teknik ska vara tillgänglig för alla. Teknik ska göra gott. Därför väljer vi kunder där vi tror att vi kan göra något bra och där vi kan göra skillnad. Det betyder också att om vi jobbar ihop så tror vi 100% på din tjänst.',
  goodTitle: 'Göra något bra',
  stabilityText:
    'Löksås ipsum plats upprätthållande från sorgliga se, denna precis från brunsås strand år dock, stig färdväg genom äng varit. Erfarenheter regn groda tidigare trevnadens hwila ännu dimma kunde mjuka sällan, oss genom tidigare annat sista stora tid plats inom.\n ',
  stabilityTitle: 'Vi står stabilt',
  valueText:
    'Vi ger dig det du behöver. Även när du tror du behöver något annat. Det kan låta kaxigt men vi vet att många företag och organisationer ofta blandar ihop det som är bråttom med det som är viktigt. Vi hjälper dig att fokusera på det som är viktigt och som skapar värde för dig och dina kunder.',
  valueTitle: 'Skapa värde',
  __typename: 'PageAboutUs',
}
