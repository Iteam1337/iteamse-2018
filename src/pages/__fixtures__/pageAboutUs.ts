export const pageAboutUs = {
  __typename: 'PageAboutUs',
  contactTitle: 'Hör av dig till oss!',
  funText:
    'Vi utvecklar din tjänst tillsammans med dig. Det blir mycket bättre så. Och roligare. För oss är det viktigt att ha kul. Därför väljer vi uppdrag med omsorg. Det blir liksom ett löfte till våra kunder: “Vi lovar att bara jobba med dig så länge vi tycker det är kul och brinner för det du gör”. Det lovar vi därför att då blir resultatet bättre.',
  funTitle: 'Ha kul',
  goodText:
    'Teknik ska vara tillgänglig för alla. Teknik ska göra gott. Därför väljer vi kunder där vi tror att vi kan göra något bra och där vi kan göra skillnad. Det betyder också att om vi jobbar ihop så tror vi 100% på din tjänst.',
  goodTitle: 'Göra något bra',
  headerImage:
    '//images.ctfassets.net/rj4r6yfcesw5/j8ng4HkakgMCyIMqeek2A/82be3a1d363b21dd29b081e76d0fdfeb/about.jpg',
  headerText1: 'Vi skapades ur en vilja att skapa värde,',
  headerText2: 'ha kul och göra något bra',
  headerTextBgColor: '#54FBAC',
  imageBleed:
    '//images.ctfassets.net/rj4r6yfcesw5/7bwz7g2GuQu8A68QmCQAMw/2c6391a40ebe2d0bee958dacf72aa174/bleed_aboutus.jpg',
  stabilityIcons: [
    '//images.ctfassets.net/rj4r6yfcesw5/2jLXgnJMdqcMIWgiYAEyq2/d88293e3899d0e65ad99754aa45f5f85/aaa_logo_web_ny.png',
    '//images.ctfassets.net/rj4r6yfcesw5/6tIo0BhFEki6Iyc68AkgiQ/98253a689d719382d5d13f4896f10941/vas-superforetagen-2016._300dppipng.png',
  ],
  stabilityText:
    'Löksås ipsum plats upprätthållande från sorgliga se, denna precis från brunsås strand år dock, stig färdväg genom äng varit. Erfarenheter regn groda tidigare trevnadens hwila ännu dimma kunde mjuka sällan, oss genom tidigare annat sista stora tid plats inom.\n ',
  stabilityTitle: 'Vi står stabilt',
  team: ['rln', 'msr'],
  valueText:
    'Vi ger dig det du behöver. Även när du tror du behöver något annat. Det kan låta kaxigt men vi vet att många företag och organisationer ofta blandar ihop det som är bråttom med det som är viktigt. Vi hjälper dig att fokusera på det som är viktigt och som skapar värde för dig och dina kunder.',
  valueTitle: 'Skapa värde',
}
