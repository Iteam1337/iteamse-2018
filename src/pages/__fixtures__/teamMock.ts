import { TEAM_QUERY } from '../../components/Team/Team'
import { teamMembers } from './teamMembers'

export const TeamQueryMock = {
  request: {
    query: TEAM_QUERY,
  },
  result: {
    data: {
      teamMembers,
    },
  },
}
