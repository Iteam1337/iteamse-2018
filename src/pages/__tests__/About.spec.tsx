import React from 'react'
import { render, wait } from 'react-testing-library'
import MockedQuery from '../../utils/test-utils/MockedQuery'
import { pageAboutUs } from '../__fixtures__/pageAboutUs'
import { TeamQueryMock } from '../__fixtures__/teamMock'
import About, { ABOUT_PAGE_QUERY } from '../About'

const mocks = [
  {
    request: {
      query: ABOUT_PAGE_QUERY,
    },
    result: {
      data: {
        pageAboutUs,
      },
    },
  },
  TeamQueryMock,
]

describe('components/About', () => {
  it('renders About', async () => {
    const { container } = render(
      <MockedQuery mocks={mocks}>
        <About />
      </MockedQuery>
    )

    await wait()

    expect(container).toMatchSnapshot()
  })
})
