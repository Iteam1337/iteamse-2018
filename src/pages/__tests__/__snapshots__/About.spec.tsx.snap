// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/About renders About 1`] = `
<div>
  <div>
    <div
      class="sc-iyvyFf hgIbDS"
    >
      <div
        class="sc-hwwEjo idbpfi"
      >
        <div
          class="sc-jtRfpW kaiGPi"
        >
          <a
            class="sc-kTUwUJ ijPJJc"
          >
            <img
              class="sc-dqBHgY kmuJpj"
              data-testid="logo"
              path="test-file-stub"
            />
          </a>
          <div
            class="sc-gxMtzJ fEeBIz"
          >
            <a
              class="sc-gzOgki dsuNbc"
              aria-current="false"
            >
              Erbjudanden
            </a>
            <a
              class="sc-gzOgki dsuNbc"
              aria-current="false"
            >
              Våra case
            </a>
            <a
              class="sc-gzOgki dsuNbc"
              aria-current="false"
            >
              Metod
            </a>
            <a
              class="sc-gzOgki dsuNbc"
              aria-current="false"
            >
              Medarbetare
            </a>
            <a
              class="sc-gzOgki dsuNbc"
              aria-current="false"
            >
              Karriär
            </a>
            <a
              class="sc-gzOgki dsuNbc"
              aria-current="false"
            >
              Om
            </a>
            <div
              class="sc-dfVpRl foeOrh"
            />
          </div>
        </div>
        <div
          class="sc-kPVwWT bZitsY"
        >
          <h1
            class="sc-kfGgVZ cZHGPc"
          >
            <span>
              Vi skapades ur en vilja att skapa värde,
              ha kul och göra något bra
            </span>
          </h1>
        </div>
      </div>
    </div>
  </div>
  <div
    class="sc-iwsKbI fwFOwW"
  >
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Skapa värde
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat bkGJgg"
            >
              <div>
                <p>
                  Vi ger dig det du behöver. Även när du tror du behöver något annat. Det kan låta kaxigt men vi vet att många företag och organisationer ofta blandar ihop det som är bråttom med det som är viktigt. Vi hjälper dig att fokusera på det som är viktigt och som skapar värde för dig och dina kunder.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Ha kul
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat bkGJgg"
            >
              <div>
                <p>
                  Vi utvecklar din tjänst tillsammans med dig. Det blir mycket bättre så. Och roligare. För oss är det viktigt att ha kul. Därför väljer vi uppdrag med omsorg. Det blir liksom ett löfte till våra kunder: “Vi lovar att bara jobba med dig så länge vi tycker det är kul och brinner för det du gör”. Det lovar vi därför att då blir resultatet bättre.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="sc-VigVT laHvOe"
    >
      <img
        class="sc-jTzLTM jKPziq"
        src="test-file-stub"
      />
    </div>
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Göra något bra
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            <div
              class="sc-htpNat bkGJgg"
            >
              <div>
                <p>
                  Teknik ska vara tillgänglig för alla. Teknik ska göra gott. Därför väljer vi kunder där vi tror att vi kan göra något bra och där vi kan göra skillnad. Det betyder också att om vi jobbar ihop så tror vi 100% på din tjänst.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <img
      class="sc-gqjmRU kaGJTv"
      src="//images.ctfassets.net/rj4r6yfcesw5/7bwz7g2GuQu8A68QmCQAMw/2c6391a40ebe2d0bee958dacf72aa174/bleed_aboutus.jpg"
    />
    <div
      class="react-reveal"
      style="opacity: 1;"
    >
      <div
        class="sc-bdVaJa fGzlqx"
        data-test="block-"
      >
        <div
          class="sc-bxivhb kgeHJJ"
        >
          <div
            class="sc-ifAKCX cFlEyZ"
          >
            <h3
              class="sc-bwzfXH kgEpTV"
            >
              Vi står stabilt
            </h3>
          </div>
          <div
            class="sc-bZQynM hmvJjY"
          >
            Löksås ipsum plats upprätthållande från sorgliga se, denna precis från brunsås strand år dock, stig färdväg genom äng varit. Erfarenheter regn groda tidigare trevnadens hwila ännu dimma kunde mjuka sällan, oss genom tidigare annat sista stora tid plats inom.
            <div
              class="sc-fjdhpX bYgbrg"
            >
              <img
                class="sc-jzJRlG gpuIau"
                alt=""
                src="//images.ctfassets.net/rj4r6yfcesw5/2jLXgnJMdqcMIWgiYAEyq2/d88293e3899d0e65ad99754aa45f5f85/aaa_logo_web_ny.png"
              />
              <img
                class="sc-jzJRlG gpuIau"
                alt=""
                src="//images.ctfassets.net/rj4r6yfcesw5/6tIo0BhFEki6Iyc68AkgiQ/98253a689d719382d5d13f4896f10941/vas-superforetagen-2016._300dppipng.png"
              />
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
